.github/workflows/ci.yml

```diff
@@ -65,4 +65,4 @@ jobs:
     - name: Install typescript
       run: npm install --save-dev typescript
 
-    - run: node_modules/.bin/tsc --strict lib/portfinder.d.ts
+    - run: node_modules/.bin/tsc --strict --lib es2015 lib/portfinder.d.ts
```

Comment:
Needed to add `--lib es2015` here as some of the dependencies that jest brings in uses es2015 features (e.g. `Set`, `Map`, etc.) and without this would get an error. Node 6+ is largely es2015 complete, so not really a compatibility risk.